scripts/build/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "build",
+  "version": "0.5.7",
```

Comment:
does it make sense to version this?

Reply:
I think some tools(I _think_ it's lerna and yarn but I'm not totally sure) complain when there isn't a version so since it doesn't really hurt anything, we might as well leave it.